demo/index.js

```diff
@@ -28,31 +32,53 @@ const StaticStorage = require('../src')({
         "accountName": "<account_name>",
         "accountKey": "<account_key>",
         "containerName": "<container_name>"
+    },
+    'gcloud': {
+        "projectId": "<project_id>",
+        "bucketName": "<bucket_name>",
+        "keyFileName": "<key_file_name>"
```

Comment:
keyFileName'e gerek yokmus. kendi makinanda `gcloud auth application-default login` bunu calistiriyorsun. kubernetes'de ise az sonra yazagim guide'i takip etmek gerekiyor:
https://cloud.google.com/container-engine/docs/tutorials/authenticating-to-cloud-platform